src/clm/commands/collapse_carbon_files.py

```diff
@@ -0,0 +1,37 @@
+from clm.functions import read_csv_file, write_to_csv_file
+
+
+def add_args(parser):
+    parser.add_argument(
+        "--carbon_files",
+        type=str,
+        nargs="+",
+        help="Input file paths for compressed carbon files",
+    )
+    parser.add_argument(
+        "--output_file",
+        type=str,
+        help="File path to save the all the compressed carbon files",
+    )
+    return parser
+
+
+def collapse_carbon_files(carbon_files, output_file):
+    carbon_files = sorted(carbon_files)
+    header_list = read_csv_file(carbon_files[0]).columns.tolist()
+    header = "".join(header_list)
```

Comment:
Don't we need some kind of delimiter here? Also, since header_list is only used at this single place, that variable can go away and this line merged into the previous one.